src/renderers/WebGLRenderer.js

```diff
@@ -2901,7 +2905,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
-	function setCubeTexture ( texture, slot ) {
+	function setTextureCube ( texture, slot ) {
```

Comment:
I left the names different because both do different things. Local `setCubeTexture` did not handle the render target case, while public `setTextureCube` did (and still does).